LDMP/data/gee_datasets.json

```diff
@@ -550,4 +550,46 @@
             "Comments": ""
         }
     }
+    "Woods Hole Research Center": {
+        "Total Carbon": {
+            "Data source": "https://data.globalforestwatch.org/datasets/aboveground-live-woody-biomass-density?selectedAttribute=shape_Length",
+            "Start year": "NA",
+            "End year": "NA",
+            "Spatial resolution": "30 m",
+            "Temporal resolution": "one time",
+            "Min Latitude": -90,
+            "Max Latitude": 90,
+            "Min Longitude": -180,
+            "Max Longitude": 180,
+            "Units": "Mg ha-1",
+            "Description": "This work generated a global-scale, wall-to-wall map of aboveground biomass (AGB) at approximately 30-meter resolution. This data product expands on the methodology presented in Baccini et al. (2012) to generate a global map of aboveground live woody biomass density (megagrams biomass ha-1) at 0.00025-degree (approximately 30-meter) resolution for the year 2000",
+            "Link to Dataset": "",
+            "GEE Dataset": "users/geflanddegradation/toolbox_datasets/forest_agb_30m_woodhole",
+            "License": "",
+            "License URL": "",
+            "Source": "https://data.globalforestwatch.org/datasets/aboveground-live-woody-biomass-density",
+            "Comments": ""
+        }
+    }
```

Comment:
You are missing a comma here - before pushing any changes makes sure you test them locally in QGIS to ensure the plugin still loads. You can do that with:

invoke plugin-install

(or with)

invoke plugin-install -f

The "-f" will be faster since it won't recompile everything every time, only those files it things has changed.

Reply:
@azvoleff when I run the 'invoke plugin-setup' and then 'invoke plugin-install' I'm getting the same message Monica reported on #255